slices/fonts-dejavu-core.yaml

```diff
@@ -0,0 +1,29 @@
+package: fonts-dejavu-core
+
+slices:
+  fonts:
+    essential:
+      - fonts-dejavu-mono_fonts
+    contents:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-lgc-sans.conf:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-lgc-serif.conf:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-sans.conf:
+      /etc/fonts/conf.avail/20-unhint-small-dejavu-serif.conf:
+      /etc/fonts/conf.avail/57-dejavu-sans.conf:
+      /etc/fonts/conf.avail/57-dejavu-serif.conf:
+      /etc/fonts/conf.avail/58-dejavu-lgc-sans.conf:
+      /etc/fonts/conf.avail/58-dejavu-lgc-serif.conf:
+
+      /etc/fonts/conf.d/20-unhint-small-dejavu-lgc-sans.conf:
+      /etc/fonts/conf.d/20-unhint-small-dejavu-lgc-serif.conf:
+      /etc/fonts/conf.d/20-unhint-small-dejavu-sans.conf:
+      /etc/fonts/conf.d/20-unhint-small-dejavu-serif.conf:
+      /etc/fonts/conf.d/57-dejavu-sans.conf:
+      /etc/fonts/conf.d/57-dejavu-serif.conf:
+      /etc/fonts/conf.d/58-dejavu-lgc-sans.conf:
+      /etc/fonts/conf.d/58-dejavu-lgc-serif.conf:
+
+      /usr/share/fonts/truetype/dejavu/DejaVuSans-Bold.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSans.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSerif-Bold.ttf:
+      /usr/share/fonts/truetype/dejavu/DejaVuSerif.ttf:
```

Comment:
Please include the copyright slice and its reference similary to what has been suggested in fontconfig-config slice.